docs/Language-specification.md

```diff
@@ -764,6 +766,8 @@ rule deps[msvc] Cc {
 }
 ```
 
+TODO: The `$` in the statement irritates me. Does it not need to be escaped?
```

Comment:
The above action is a blatantly incorrect copy-paste. I'll replace it with the proper variables.